subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -92,10 +92,10 @@ abstract class AbstractSmokeTest extends Specification {
         static springDependencyManagement = "1.1.0"
 
         // https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-gradle-plugin
-        static springBoot = "2.7.11"
+        static springBoot = "3.1.1"
```

Comment:
That one might be a challenge since it requires Java 17

Reply:
Yep, you are right, I'll revert this